Jenkinsfile

```diff
@@ -58,6 +58,8 @@ END
         try{
             postStatus(getPendingStatus("Running the linters..."))
             sh "docker-compose run --rm -T  webpack npm run eslint"
+            sh "docker-compose run --rm eventkit black --check /home/eventkit/miniconda3/envs/eventkit-cloud/lib/python3.6/site-packages/eventkit_cloud"
```

Comment:
I think you're going to have to explicitly point Black to the config file since it isn't in a default location using --config.  https://black.readthedocs.io/en/stable/pyproject_toml.html#where-black-looks-for-the-file